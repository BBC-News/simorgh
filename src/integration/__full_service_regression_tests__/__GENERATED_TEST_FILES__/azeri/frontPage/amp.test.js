/**
 * @service azeri
 * @pathname /azeri
 *
 * THIS TEST SUITE WAS GENERATED BY A BUILD SCRIPT. DO NOT EDIT THIS FILE.
 */

import runCrossPlatformTests from '../../../../pages/frontPage/crossPlatformTests';
import runAmpTests from '../../../../pages/frontPage/ampTests';
describe(platform.toUpperCase(), () => {
  describe(pageType, () => {
    runCrossPlatformTests();
    runAmpTests();
  });
});
