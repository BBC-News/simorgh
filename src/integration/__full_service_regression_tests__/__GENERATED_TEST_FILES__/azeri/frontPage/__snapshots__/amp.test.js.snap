// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP  A11y I can see the skip to content link 1`] = `"Mətnə keçid"`;

exports[`AMP  AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
}
`;

exports[`AMP  AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP  Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=6&p=azeri.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:cps:3a2d4038-19a9-e1e2-e050-380a4e8076f3]&x2=[amp]&x3=[news-azeri]&x4=[az]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[index-home]&x8=[simorgh]&x9=[Xəbərlər+-+BBC+News+Azərbaycanca]&x11=[1970-01-18T00:43:51.886Z]&x12=[1970-01-19T02:08:45.487Z]",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP  Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/azeri"`;

exports[`AMP  Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. BBC kənar saytların məzmununa məsul deyil. Bizim kənar keçidlərə dair yanaşmamız barədə oxuyun."`;

exports[`AMP  Header I can see the branding 1`] = `"BBC News, Azərbaycanca"`;

exports[`AMP  SEO Canonical link 1`] = `"http://localhost:7080/azeri"`;

exports[`AMP  SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP  SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP  SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP  SEO Heading level 1 1`] = `"BBC News, Azərbaycanca - Xəbərlər"`;

exports[`AMP  SEO Lang attribute 1`] = `"az"`;

exports[`AMP  SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/azeri",
    "@type": "WebPage",
    "name": "Xəbərlər",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
      "width": 1024,
    },
    "name": "BBC News Azərbaycanca",
    "publishingPrinciples": "https://www.bbc.com/azeri/institutional-49283479",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
  "url": "http://localhost:7080/azeri",
}
`;

exports[`AMP  SEO OG description 1`] = `"BBC News Azərbaycanca öz səhifələrində Azərbaycan və dünyadan bu günün xəbərləri və şərhlərini təqdim edir. Visit BBC News Azeri for the latest from Azerbaijan, Armenia, Georgia."`;

exports[`AMP  SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png"`;

exports[`AMP  SEO OG image alt 1`] = `"BBC News Azərbaycanca"`;

exports[`AMP  SEO OG locale 1`] = `"az-AZ"`;

exports[`AMP  SEO OG site name 1`] = `"BBC News Azərbaycanca"`;

exports[`AMP  SEO OG title 1`] = `"Xəbərlər, Qaynar Xəbərlər, Analiz, Video - BBC News Azərbaycanca"`;

exports[`AMP  SEO OG type 1`] = `"website"`;

exports[`AMP  SEO OG url 1`] = `"http://localhost:7080/azeri"`;

exports[`AMP  SEO Page title 1`] = `"Xəbərlər, Qaynar Xəbərlər, Analiz, Video - BBC News Azərbaycanca"`;

exports[`AMP  SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP  SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP  SEO Twitter creator 1`] = `"@bbcazeri"`;

exports[`AMP  SEO Twitter description 1`] = `"BBC News Azərbaycanca öz səhifələrində Azərbaycan və dünyadan bu günün xəbərləri və şərhlərini təqdim edir. Visit BBC News Azeri for the latest from Azerbaijan, Armenia, Georgia."`;

exports[`AMP  SEO Twitter image alt 1`] = `"BBC News Azərbaycanca"`;

exports[`AMP  SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png"`;

exports[`AMP  SEO Twitter site 1`] = `"@bbcazeri"`;

exports[`AMP  SEO Twitter title 1`] = `"Xəbərlər, Qaynar Xəbərlər, Analiz, Video - BBC News Azərbaycanca"`;
